Armstrong.WebClient/Program.cs

```diff
@@ -6,6 +6,8 @@
 builder.RootComponents.Add<App>("#app");
 builder.RootComponents.Add<HeadOutlet>("head::after");
 
-builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+builder.Services.AddScoped(sp => new HttpClient { 
+    BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) 
+});
 
-await builder.Build().RunAsync();
+await builder.Build().RunAsync();
```

Comment:
Why do you have this rare of (end of file)

Reply:
Fixed